codegen/src/main/java/software/amazon/awssdk/codegen/model/service/OperationContextParam.java

```diff
@@ -18,13 +18,13 @@
 import com.fasterxml.jackson.core.TreeNode;
 
 public class OperationContextParam {
-    private TreeNode value;
+    private TreeNode path;
 
-    public TreeNode getValue() {
-        return value;
+    public TreeNode getPath() {
+        return path;
     }
 
-    public void setValue(TreeNode value) {
-        this.value = value;
+    public void setPath(TreeNode path) {
```

Comment:
Path is always a string, which we can map directly without having to use a `TreeNode` instance here. As in 

```
package software.amazon.awssdk.codegen.model.service;

public class OperationContextParam {
    private String path;

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }
}
```
This will also simplify the code that uses those values.

```